.github/workflows/build.yaml

```diff
@@ -22,6 +22,15 @@ jobs:
           scope: "mozilla"
       - run: npm install -g @microsoft/rush
       - run: rush update
-      - run: rush install        
+      - run: rush install
       - run: rush build
       - run: rush test
+
+      - name: Deploy firebase functions to staging
+        run: |
+            cp web-platform/firebase/firebase.config.rally-web-stage.json web-platform/firebase.config.json
+            cp web-platform/firebase/firestore.rules web-platform/firestore.rules
+            cp web-platform/firebase/firebase.json web-platform/firebase.json
+            sed -i 's/\.\.\/functions/functions/g' web-platform/firebase.json
+            echo ${{ secrets.GCLOUD_SERVICE_KEY }} | base64 --decode --ignore-garbage > "$HOME"/gcloud.json
+            GOOGLE_APPLICATION_CREDENTIALS="$HOME"/gcloud.json web-platform/functions/node_modules/.bin/firebase deploy --only auth,functions,firestore --project ${{secrets.PROJECT_ID_STAGE}} -c web-platform/firebase.json
```

Comment:
I'm not sure `auth` is a target for the `--only` param, having trouble finding good docs on the list of targets... I don't think there's anything to do there. I believe `firestore` will deploy the `firebase/firestore.rules` file so we should make sure the path to that is correct in `firebase.json`

Reply:
Removed auth.